Resources/Prototypes/Entities/Mobs/Player/silicon.yml

```diff
@@ -545,27 +545,34 @@
       borg_brain:
         - PositronicBrain
       borg_module:
-        - BorgModuleTool
-        - BorgModuleOperative
-        - BorgModuleSyndicateWeapon
+        - BorgModuleSyndicateTool # Harmony Change
+        - BorgModuleBulldog # Harmony Change
   - type: ItemSlots
     slots:
       cell_slot:
         name: power-cell-slot-component-slot-name-default
         startingItem: PowerCellHyper
 
 - type: entity
-  id: PlayerBorgSyndicateSaboteurGhostRole
-  parent: PlayerBorgSyndicateSaboteurBattery
-  suffix: Ghost role
```

Comment:
- Same, leave the old entity but comment the whole of it out and mark it as a harmony change.
- Define new entities in `Resources/Prototypes/_Harmony/Entities/Mobs/Player/silicon.yml` instead of putting them here, as they can and should be moved.